src/ucan.js

```diff
@@ -88,14 +93,16 @@ export async function sign(payload, keypair) {
   return {
     header,
     payload,
-    signature: encodedSig,
+    signature: sig,
     jwt: encodedHeader + '.' + encodedPayload + '.' + encodedSig,
   }
 }
 
 /**
  * @param {string} encodedUcan
  * @param {import('./types').ValidateOptions} [options]
+ * 
+ * @returns {Promise<import('./types').Ucan>}
  */
 export async function validate(encodedUcan, options) {
```

Comment:
```suggestion
export async function validate(encodedUcan, options = {}) {
```

Reply:
we need to have an object default or else the merge below will fail if options are undefined

Reply:
Good catch! 👍